src/main/java/com/synopsys/arc/jenkins/plugins/ownership/extensions/item_ownership_policy/PreserveOwnershipPolicy.java

```diff
@@ -0,0 +1,43 @@
+package com.synopsys.arc.jenkins.plugins.ownership.extensions.item_ownership_policy;
```

Comment:
It would be preferable to keep the new code in the `org.jenkinsci.plugins.ownership.config` package